open-metadata-resources/open-metadata-tutorials/building-egeria-tutorial/README.md

```diff
@@ -0,0 +1,38 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+# Downloading and Building ODPi Egeria Tutorial
+
+ODPi Egeria is a an open source project that is delivered as source code as well as libraries
+stored in the [Maven Central Repository](https://search.maven.org/).
+
+This tutorial will guide you through the process of downloading the ODPi Egeria source
+code from [GitHub](https://github.com/) and building it so that you can run it
+on your local machine.
+
+## Prerequisite tasks
+
+* Ensure you have a Java compiler at Version 8 (Update 151) or above installed on your machine.
+Check the version of Java you have with the command `java -version`
+(You can download Java from [AdoptOpenJDK](https://adoptopenjdk.net/) and select `OpenJDK 8 (LTS)`
+and the `HotSpot` version for your operating system.  You need the JDK rather than the JRE.)
+* Ensure you have [Apache Maven](https://maven.apache.org/) 3.5.0 or above installed.
+This is used to orchestrate the ODPi Egeria build.
+* Ensure you have `git` installed.  On MacOS, this is part of the `Xcode` suite.
+For other operating systems there are [various choices](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+
+
+## Tutorial tasks
+
+* [Downloading the Egeria source from GitHub](task-downloading-egeria-source.md)
```

Comment:
```suggestion
1. [Downloading the Egeria source from GitHub](task-downloading-egeria-source.md)
```

Reply:
Don't understand this change - seems to be switching the last bullet in list ot a number

Reply:
Meant to change the bullet list into a numbered sequence, as the ordering seemed important — not sure what happened to the other edits